packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts

```diff
@@ -173,10 +173,9 @@ function computeEntryTracingInfo(entry: RumPerformanceResourceTiming, configurat
 }
 
 function toPerformanceEntryRepresentation(entry: RumPerformanceEntry): PerformanceEntryRepresentation {
-  if (supportPerformanceEntry() && entry instanceof PerformanceEntry) {
-    entry.toJSON()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏I wonder if `toJSON` is actually useful here, since we didn't even use its returned value in the original code. Maybe we could just remove it?

Reply:
fair point, probably not.
We should probably drop `PerformanceEntryRepresentation` and use directly `PerformanceEntry` then (in the next major).